.travis.yml

```
@@ -11,9 +11,13 @@ rvm:
 script:
 - bundle exec rspec
 
-deploy:
-  provider: script
-  script: ./bin/travis_deploy.sh
-  on:
-    branch: main
-    condition: type != pull_request
+jobs:
+  include:
+    - stage: gem release
+      rvm: 3.2.1
+      deploy:
+        provider: script
+        script: ./bin/travis_deploy.sh
+        on:
+          branch: main
+          condition: type != pull_request
```

Not strictly necessary, but I think this will skip the deploy job on PRs, and skip rspec on deploy:
```suggestion
      if: type != pull_request && branch = main
      script: skip
      deploy:
        provider: script
        script: ./bin/travis_deploy.sh
```